site/en/docs/extensions/mv3/getstarted/tut-tabs-manager/index.md

```diff
@@ -0,0 +1,400 @@
+  <figcaption>
+  Tabs Manager extension
+  </figcaption>
+</figure>
+
+In this guide, we’re going to explain how to do the following:
+
+- Create an extension popup using the [Action][api-action] API.
+- Query for specific tabs using the [Tabs][api-tabs] API.
+- Preserve user privacy through narrow host permissions.
+- Change the focus of the tab.
+- Move tabs to the same window and group them.
+- Rename tabs group using the [TabGroups][api-tabgroups] API.
+
+## Before you start {: #prereq }
+
+This guide assumes that you have basic web development experience. We recommend checking out
+[Development Basics][doc-dev-basics] for an introduction to the extension development workflow.
+
+## Build the extension {: #build }
+
+To start, create a new directory called `tabs-manager` that will hold the extension's files. If you
+prefer, you can download the complete source code on [Github][github-tabs-manager].
+
+### Step 1: Add the extension data and icons {: #step-1 }
+
+Create a file called `manifest.json` and include the following code:
+
+{% Label %}manifest.json:{% endLabel %}
+
+```json
+{
+  "manifest_version": 3,
+  "name": "Tab Manager for Chrome Dev Docs",
+  "version": "1.0",
+  "icons": {
+    "16": "images/icon-16.png",
+    "32": "images/icon-32.png",
+    "48": "images/icon-48.png",
+    "128": "images/icon-128.png"
+  },
+  ...
+}
+```
+
+These manifest keys are explained in more detail in the [Reading time
+tutorial][tut-reading-time-step1]. 
+
+Create an `images` folder and place the icons inside. You can download the icons
+[here][github-tabs-manager-icons].
+
+### Step 2: Create and style the popup {: #step-2 }
+
+The [Action][api-action] API controls the extension action (toolbar icon). When the user clicks on
+the extension action, it will either run some code or open a popup, like in this case. Start by
+declaring the popup in the manifest:
+
+{% Label %}manifest.json:{% endLabel %}
+
+```json
+{
+  ...
+  "action": {
+    "default_popup": "popup.html"
+  },
+  ...
+}
+```
+
+A popup is similar to a web page with one exception: it can't run inline JavaScript. Create a popup
+HTML file and add the following code:
+
+{% Label %}popup.html:{% endLabel %}
+
+```html
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./popup.css" />
+  </head>
+  <body>
+    <template id="li_template">
+      <li>
+        <a>
+          <h3 class="title">Tab Title</h3>
+          <p class="pathname">Tab Pathname</p>
+        </a>
+      </li>
+    </template>
+
+    <h1>Google Dev Docs</h1>
+    <button>Group Tabs</button>
+    <ul></ul>
+
+    <script src="./popup.js" type="module"></script>
+  </body>
+</html>
+```
+
+{% Aside %}
+
+💡 **TIP**: You can use [top level await][mdn-top-level] by adding `type="module”` to the script
```

Comment:
What is this and why would I want to use it?

Reply:
In practice, you can use the await keyword outside of async functions. See a more [in-depth explanation](https://v8.dev/features/top-level-await).
  
I think most javascript developers would be familiar with this concept. But we can check with @dotproto to see if adding more context is necessary.